pytket/tests/qasm_test_files/test1.qasm

```diff
@@ -4,6 +4,7 @@ include "qelib1.inc";
 qreg q[4];
 rz(1.5*pi) q[3];
 rx(0.0375*pi) q[3];
+rxx(0.0375*pi) q[3];
```

Comment:
This is a two-qubit operation so needs two qubits to be specified. (The test should be modified accordingly.)

Reply:
Done.